engine/flink/tests/src/test/scala/pl/touk/nussknacker/devmodel/SqlKafkaPingPongTest.scala

```diff
@@ -0,0 +1,92 @@
+package pl.touk.nussknacker.devmodel
+
+import com.typesafe.config.{Config, ConfigFactory}
+import io.circe.Json
+import pl.touk.nussknacker.defaultmodel.{FlinkWithKafkaSuite, TopicConfig}
+import pl.touk.nussknacker.engine.api.component.ComponentDefinition
+import pl.touk.nussknacker.engine.api.process.ProcessObjectDependencies
+import pl.touk.nussknacker.engine.build.ScenarioBuilder
+import pl.touk.nussknacker.engine.flink.table.{SqlComponentProvider, TableComponentProvider}
+import pl.touk.nussknacker.engine.kafka.KafkaTestUtils.richConsumer
+import pl.touk.nussknacker.engine.schemedkafka.KafkaUniversalComponentTransformer
+import pl.touk.nussknacker.engine.schemedkafka.schemaregistry.SchemaVersionOption
+import pl.touk.nussknacker.engine.spel
+import TestData._
+
+import java.nio.file.Files
+
+class SqlKafkaPingPongTest extends FlinkWithKafkaSuite {
+
+  import spel.Implicits._
+
+  private val testNameTopicPart: String = "sql-pp"
+  private val topicNaming1: String      = s"$testNameTopicPart.test1"
+  private lazy val inputTopicNameTest1  = TopicConfig.inputTopicName(topicNaming1)
+
+  private lazy val sqlTablesConfig =
+    s"""
+       |CREATE TABLE testTable (
+       |   someInt     INT,
+       |   someString  STRING
+       | ) WITH (
+       |  'connector' = 'kafka',
+       |  'topic' = '$inputTopicNameTest1',
+       |  'properties.bootstrap.servers' = '${kafkaServer.kafkaAddress}',
+       |  'properties.group.id' = 'someConsumerGroupId',
+       |  'scan.startup.mode' = 'earliest-offset',
+       |  'format' = 'json'
+       |);""".stripMargin
+
+  private lazy val sqlTablesDefinitionFilePath = {
```

Comment:
Instead of creating the temp file, we can replace `SqlFileDataSourceConfig.sqlFilePath` with URL and use pass a resources location. WDYT?

Reply:
I wanted to use a path relative to the base path since it's easier to pass this path through env variables in designer configuration and in flink container. For this test I have to compose a file with content depending on kafka address, so I don't think I can do it like that.

Reply:
done :+1: